packages/streams/timestamp-throttle.js

```diff
@@ -40,9 +49,13 @@ TimestampThrottle.prototype._transform = function (msg, encoding, done) {
   const msgMillis = this.getMilliseconds(msg)
   if (msgMillis < this.lastMsgMillis) {
     this.offsetMillis = new Date().getTime() - msgMillis
+    this.firstOffsetMillis = this.offsetMillis
```

Comment:
`firstOffsetMillis` is always the same as as `offsetMillis`, so it is not really needed?